### PR TITLE
docs: clean up examples

### PR DESCRIPTION
**Pull request by harrysolovay**
Cleaning up examples for consumption by docs.capi.dev.

### CODE REVIEW

File: examples/sign/offline.eg.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ const hash = await SignedExtrinsicRune
 // Ensure the extrinsic has been finalized.
 $.assert($.str, hash)
 
-// egdoc-ignore-start
+// egts-ignore-start
```
Comment (tjjfvi) [inline examples/sign/offline.eg.ts:39]:
```suggestion
// hide-start
```

File: examples/sign/offline.eg.ts [hunk 2]
```diff
@@ -36,8 +36,8 @@ const hash = await SignedExtrinsicRune
...
 // The following noop is solely for explanation. Swap this out with your
 // own signed-hex-representation-consuming code.
 function save(_hex: string) {}
-// egdoc-ignore-end
+// egts-ignore-end
```
Comment (tjjfvi) [inline examples/sign/offline.eg.ts:43]:
```suggestion
// hide-end
```

File: import_map.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "scopes": {
     "examples/": {
       "capi": "./mod.ts",
-      "capi/": "./",
+      "capi/patterns/": "./patterns/",
```
Comment (tjjfvi) [inline import_map.json:8]: Why this change?

Comment (harrysolovay) [reply import_map.json:8]: Otherwise we get a warning on importing from `"capi"` (wants to redirect to `"capi/patterns/..."`)

**Review by tjjfvi — APPROVED** [review]
